src/mame/dai/dai_m.cpp

```diff
@@ -210,5 +211,5 @@ uint8_t dai_state::amd9511_r()
 
 void dai_state::amd9511_w(offs_t offset, uint8_t data)
 {
-	logerror ("Writing to AMD9511 math chip, %04x, %02x\n", offset, data);
+	LOG("Writing to AMD9511 math chip, %04x, %02x\n", offset, data);
 }
```

Comment:
This log message is a placeholder for unimplemented functionality – it should be on by default.

Reply:
Switched it back to logerror.